apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -169,22 +172,35 @@ const InlineAudio = React.createClass({
   render: function () {
     if (this.props.textToSpeechEnabled && !this.state.error && this.isLocaleSupported() && this.getAudioSrc()) {
       return (
-        <div className="inline-audio">
-          <div style={[styles.button, styles.volumeButton]}>
-            <img style={styles.buttonImg} src={this.props.assetUrl("media/common_images/volume.png")} />
+        <div
+          className="inline-audio"
+          style={(this.props.style && this.props.style.wrapper) ? this.props.style.wrapper : null}
```

Comment:
I believe you can just do `style={this.props.style && this.props.style.wrapper}`, here and elsewhere